tests/phpunit/tests/post/thumbnails.php

```diff
@@ -408,6 +408,41 @@ public function test_post_thumbnail_size_filter_complex( $which_post, $expected
 		$this->assertSame( $expected, $result );
 	}
 
+	/**
+	 * @ticket 57490
+	 */
+	public function test_get_the_post_thumbnail_includes_loading_lazy() {
+		set_post_thumbnail( self::$post, self::$attachment_id );
+
+		$html = get_the_post_thumbnail( self::$post );
+		$this->assertStringContainsString( ' loading="lazy"', $html );
+	}
+
+	/**
+	 * @ticket 57490
+	 */
+	public function test_get_the_post_thumbnail_respects_passed_loading_attr() {
+		set_post_thumbnail( self::$post, self::$attachment_id );
+
+		$html = get_the_post_thumbnail( self::$post, 'post-thumbnail', array( 'loading' => 'eager' ) );
+		$this->assertStringContainsString( ' loading="eager"', $html, 'Attributes array with loading=eager was overwritten.' );
```

Comment:
The `$message` parameter should explain what went wrong.

I.e.
```suggestion
		$this->assertStringContainsString( ' loading="eager"', $html, 'Attributes array with loading=eager was not present in IMG tag.' );
```

Reply:
Hmm, what would have gone wrong in that case is that the attributes array was overwritten though, I think your suggestion is misleading in that sense. The main reason this needs a message is because there are 2 assertions in 1 test, so I wanted to focus primarily on what is different between the assertions. Maybe we could say "loading=eager was not present in img tag because attributes array with loading=eager was overwritten.", but that may be a bit long. WDYT?

Reply:
Thanks @felixarntz, The message you have suggested look good to me. I can't find any official document for `$message` length, so it's permissible to use it.

Check https://make.wordpress.org/core/handbook/testing/automated-testing/writing-phpunit-tests/#using-assertions for the unit test for `Tests_Foo` class; they also use long messages.

Reply:
@mukeshpanchal27 Updated!